File: M3104/tomcat-etud/webapps/gpsapp/WEB-INF/src/db/Coordinates.java
package db;


public class Coordinates
{
    private int journey_id;
    private int journey_pos;
    private double latitude;
	private double longitude;

    public Coordinates(int journey_id, double latitude, double longitude) {
        this.journey_id = journey_id;
        this.journey_pos = journey_pos;
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public int getJourney_id() {
        return journey_id;
    }

    public void setJourney_id(int journey_id) {
        this.journey_id = journey_id;
    }

    public int getJourney_pos() {
        return journey_pos;
    }

    public void setJourney_pos(int journey_pos) {
        this.journey_pos = journey_pos;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    @Override
    public String toString() {
        String ret = "--- Coordinates ---";
        ret += "\nJOURNEY_ID = " + journey_id;
        ret += "\nJOURNEY_POS = " + journey_pos;
        ret += "\nLATITUDE = " + latitude;
        ret += "\nLONGITUDE" + longitude;
        ret += "\n---------------\n";
        return ret;
    }
}

File: M3104/PHP/my_gps_web_app/model/Coordinates.php
<?php

class Coordinates {
    private $journey_id;
    private $journey_pos;
    private $latitude;
    private $longitude;

    public function getJourneyId() {
        return $this->journey_id;
    }

    public function setJourneyId($newJourneyId) {
        $this->journey_id = $newJourneyId;
    }

    public function getJourneyPos() {
        return $this->journey_pos;
    }

    public function setJourneyPos($newJourneyPos) {
        $this->$journey_pos = $newJourneyPos;
    }

    public function getLatitude() {
        return $this->latitude;
    }

    public function setLatitude($newLatitude) {
        $this->latitude = $newLatitude;
    }

    public function getLongitude() {
        return $this->longitude;
    }

    public function setLongitude($newLongitude) {
        $this->longitude = $newLongitude;
    }
}

?>
